browse/config.py

```diff
@@ -132,6 +132,13 @@ class Settings(arxiv_base.Settings):
     `./testing/data/` for testing data. Must end with a /
     """
 
+    REASONS_FILE_PATH: str = "DEFAULT"
+    """Full path to `reasons.json` file.
```

Comment:
at some point we'll want a page for the EUST to be able to curate this list.

Reply:
Ya, it should go in the DB.